server/bin/test_data_importer.py

```diff
@@ -0,0 +1,706 @@
+#!/usr/bin/env python
+from __future__ import print_function, division, absolute_import
+
+"""
+Injects synthetic data from one or more JSON files into Candlepin for testing
+"""
+
+import datetime
+import logging
+import json
+import os
+import random
+import re
+import requests
+import string
+import sys
+import time
+
+from optparse import OptionParser
```

Comment:
The `optparse` module is considered as obsolete. Please use `argparse`.

Reply:
Done